lib/mix/lib/mix/tasks/profile.fprof.ex

```diff
@@ -190,8 +195,22 @@ defmodule Mix.Tasks.Profile.Fprof do
       fun.()
     end
 
-    {:ok, tracer} = :fprof.profile(:start)
-    return_value = :fprof.apply(fun, [], tracer: tracer)
+    {return_value, file_to_remove} =
+      if Keyword.get(opts, :trace_to_file, false) do
+        trace_file = Path.join([System.tmp_dir!(), "fprof_trace_#{System.unique_integer()}"])
```

Comment:
System.unique_integer can conflict across instance restarts because it resets the VM. Would this be better?

```suggestion
        trace_file = Path.join(System.tmp_dir!(), "fprof_trace_#{System.os_time()}")
```

?

Reply:
I think its a good idea